src/main/java/seedu/bankwithus/user/Transaction.java

```diff
@@ -0,0 +1,46 @@
+package seedu.bankwithus.user;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Transaction {
+    private String accountName;
+    private String amount;
+
+    //Take Note of this date is in LocalDate format, but not a String!
+    private LocalDate date;
+    private String type;
+
+    public Transaction(String accountName, String type, String amount, LocalDate date) {
+        this.accountName = accountName;
+        this.type = type;
+        this.amount = amount;
+        this.date = LocalDate.now();
```

Comment:
Won't line 18 cause a ton of bugs when parsing the transaction save file? I tot the line is supposed to be this.date = date;.

Reply:
Oh yes! I'll change it now.